src/main/scala/rocketchip/BaseTop.scala

```diff
@@ -67,7 +67,8 @@ abstract class BaseTopModule[+L <: BaseTop, +B <: BaseTopBundle](
   val io: B = b
 
   val coreplex = p(BuildCoreplex)(outer.c, p)
-  val coreplexIO = coreplex.io
+  val coreplexIO = Wire(new BaseCoreplexBundle(outer.c)(p){})
```

Comment:
I tried doing that at first, but that requires every *CoreplexBundle class to explicitly define a cloneType method. My thought process was that if you have extra ports, you should handle those yourself. I guess it's fine to require a cloneType method for every CoreplexBundle.